plugins/synaptics-rmi/fu-synaptics-rmi-v7-device.c

```diff
@@ -331,6 +416,14 @@ fu_synaptics_rmi_v7_device_write_partition(FuSynapticsRmiDevice *self,
 			return FALSE;
 		fu_progress_step_done(progress);
 	}
+	if (!fu_synaptics_rmi_v7_device_write_partition_signature(self,
+								  firmware,
+								  id,
+								  partition_id,
+								  fu_progress_get_child(progress),
```

Comment:
I don't think this is going to work, I think you need to split up the phases with a new `fu_progres_set_id()` block and some new steps. If you push this branch to the origin remote I can push a fixup if that helps.

Reply:
Hi do I need permission to push to origin?

Reply:
You're in the commit list, so just push something like wip/vhuag/linux-add-signature-20230109

Reply:
Hi I can't push due to permission so I remove the progress tracking inside the signature function, normally the chunks->len is 1.